src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -30,23 +30,42 @@ export const InviteMemberDialogContent = ({
   groupId,
 }: IInviteMemberDialogContentProps) => {
   const [isLinkButtonClicked, setIsLinkButtonClicked] = useState(false);
+  const [isHash, setHash] = useState('');
```

Comment:
You shouldn't use `is` for a string state.

https://github.com/kettanaito/naming-cheatsheet#is